src/periodic_table.py

```diff
@@ -0,0 +1,359 @@
+METAL = 'Metal'
+NON_METAL = 'Non Metal'
+METALLOID = 'Metalloid'
+
+PERIODIC_TABLE = {
+    '1': {'atomic_num': 1, 'symbol': 'H', 'name': 'Hydrogen', 'group': 1, 'period': 1, 'atomic_weight': 1.008,
+          'density': 8.988e-05, 'melting_point': 14.01, 'boiling_point': 20.28, 'heat_capacity': 14.304,
+          'electronegativity': 2.2, 'precise': True, 'type': NON_METAL},
+    '2': {'atomic_num': 2, 'symbol': 'He', 'name': 'Helium', 'group': 18, 'period': 1, 'atomic_weight': 4.002602,
+          'density': 0.0001785, 'melting_point': None, 'boiling_point': 4.22, 'heat_capacity': 5.193,
```

Comment:
Need to make note about electronegativity set to None to use in Element and Compound classes when comparing Elements